build.sbt

```diff
@@ -108,7 +108,7 @@ val yaml = project
 lazy val dependencies =
   new {
     val dxCommonVersion = "0.8.0"
-    val dxApiVersion = "0.9.0"
+    val dxApiVersion = "0.10.1-SNAPSHOT"
```

Comment:
Is this supposed to be set to the current SNAPSHOT version, or the last released version? @jdidion

Reply:
I think it should be the current SNAPSHOT version?

Reply:
yes, the current snapshot version